modules/ext.tabberNeue.init/ext.tabberNeue.init.less

```diff
@@ -1,72 +1,92 @@
-/**
- * Critial rendering styles
- *
- * Since ext.tabberNeue is loaded a while after page load,
- * inline styles are needed to create an inital state and
- * avoid potential layout shifts. This should be kept as
- * small as possible.
```

Comment:
I feel that these comments in the beginning can be kept. They seems helpful to newcomers to understand the purpose of this file.

Reply:
I'll add the header back and reword it a bit. There are no longer initial state/skeleton screen anymore because we're rendering the header in PHP as well.

Reply:
Addressed in [27e05c7](https://github.com/StarCitizenTools/mediawiki-extensions-TabberNeue/commit/27e05c72a6547285467b175e13f0525f2e5304d1)